modules/installation-configuration-parameters.adoc

```diff
@@ -195,6 +221,185 @@ endif::[]
 
 |====
 
+[discrete]
+[id="installation-configuration-parameters-network_{context}"]
+== Network configuration parameters
+
+You can customize your installation configuration based on the requirements of your existing network infrastructure. For example, you can expand the IP address block for the cluster network or provide different IP address blocks than the defaults.
+
+ifndef::bare[]
+Only IPv4 addresses are supported.
+endif::bare[]
+ifdef::bare[]
+If you use the OVN-Kubernetes cluster network provider, both IPv4 and IPv6 address families are supported.
+
+If you use the OpenShift SDN cluster network provider, only the IPv4 address family is supported.
+
+If you configure your cluster to use both IP address families, you must specify IPv4 and IPv6 addresses in the same order for all network configuration parameters. For example, in the following configuration IPv4 addresses are listed before IPv6 addresses.
+
+[source,yaml]
+----
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd00:10:128::/56
+    hostPrefix: 64
+  serviceNetwork:
+  - 172.30.0.0/16
+  - fd00:172:16::/112
+----
+endif::bare[]
+
+.Network parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`networking`
+|The configuration for the network for the cluster.
```

Comment:
We have other instances of "cluster network" so is this a candidate for "The configuration for the cluster network."?

Reply:
@mikemckiernan, mostly true, but `machineNetwork` is sort of adjacent, so this is weird both ways.